ci-operator/config/operator-framework/community-operators/operator-framework-community-operators-master.yaml

```diff
@@ -0,0 +1,27 @@
+base_images:
+  cli:
+    name: "4.6"
+    namespace: ocp
+    tag: cli
+build_root:
+  project_image:
+    dockerfile_path: scripts/ci/Dockerfile.ci-operator
+resources:
+  '*':
+    requests:
+      cpu: 1000m
+      memory: 4Gi
+tag_specification:
+  name: "4.6"
+  namespace: ocp
```

Comment:
What are your expectations about the installed cluster versions and stability? The `tag_specification` stanza means that `ci-operator` will assemble an ephemeral release (in this case, 4.6) from the content of the OCP integration imagestream (basically, this corresponds to HEADs of all 4.6 branches). These releases may be more unstable/unreliable than the releases vetted by the release controller, or even an already-released-release (what OCP customers have). If your goal is to provide stable feedback to operator authors and not wear them down with running on OCP integration releases, you should consider using a ` releases` stanza (https://steps.ci.openshift.org/help/ci-operator#release) that allows you to specify what kind of "existing" release you would like to use. It seems to me that a nightly, or even a released image should be a better choice than running on integration imagestream OCP.